lib/src/configuration.dart

```diff
@@ -162,7 +162,8 @@ abstract class Configuration implements Finalizable {
     String? fifoFilesFallbackPath,
     String? path,
     SyncErrorHandler syncErrorHandler = defaultSyncErrorHandler,
-    SyncClientResetErrorHandler syncClientResetErrorHandler = const ManualSyncClientResetHandler(_defaultSyncClientResetHandler),
+    SyncClientResetErrorHandler syncClientResetErrorHandler = const RecoverOrDiscardSyncClientResetHandler(_defaultSyncClientResetHandler),
+    ClientResyncMode clientResyncMode = ClientResyncMode.recoverOrDiscard,
```

Comment:
This should not be exposed.

Reply:
Oh, right. What about having only a getter of `clientResyncMode`?

Reply:
OK, I will change it.